### PR TITLE
Expose and test SystemDrawingSection

### PR DESCRIPTION
**Pull request by hughbe**
This was discussed in #20712

### CODE REVIEW

**Comment by mellinoe** [comment]
@weshaggard Could you verify that adding the extra build configuration for System.Configuration.ConfigurationManager is valid / the right thing to do here?

Code changes LGTM.

**Comment by mellinoe** [comment]
@ericstj Could you double-check the build configuration change? Wes is out for the week.

**Comment by ericstj** [comment]
I'm not sure I'd put anything above System.Configuration.ConfigurationManager in the stack.  I'd sooner put SystemDrawingSection in S.C.ConfigurationManager than add this dependency.  /cc @JeremyKuhne 

Ignoring that for a moment, you shouldn't need the new configuration of S.C.CM even with the current factoring.  You didn't make the ref significantly different (ifdefs) and I don't think you were hitting an impl cycle since nothing in netstandard would bring in Drawing.Common or Configuration into the closure.

**Comment by mellinoe** [comment]
Moving the type to S.C.CM sounds reasonable. It's similar to what we've been doing for TypeConverter.

> Ignoring that for a moment, you shouldn't need the new configuration of S.C.CM even with the current factoring. You didn't make the ref significantly different (ifdefs) and I don't think you were hitting an impl cycle since nothing in netstandard would bring in Drawing.Common or Configuration into the closure.

(I think we will probably move the type, so this is mostly moot) Is there a `netstandard` facade we can reference in order to make this work, then? The only problem was that System.Drawing.Common couldn't reference ConfigurationManager because it was referencing `netstandard` in turn.

**Comment by mellinoe** [comment]
@hughbe Let's go ahead and move this over into the ConfigurationManager library itself -- it doesn't have any dependencies on other Drawing types so it should be pretty straightforward.

**Comment by hughbe** [comment]
Sure - I updated the PR to move it to ConfigurationManager

**Comment by mellinoe** [comment]
@hughbe I believe this needs a little fixing up. I pulled down your branch and made the fixes I think are needed:

https://github.com/mellinoe/corefx/commit/d52a0b6a062d603a641a74169449e5a258d9f0d8

@JeremyKuhne Do you have any input on this? Should new `ConfigurationSection` types go into ConfigurationManager?

**Comment by JeremyKuhne** [comment]
I'm fine with adding new ConfigurationSection items to ConfigurationManager as long as we don't introduce additional dependencies under ConfigurationManager

File: src/System.Configuration.ConfigurationManager/ref/System.Configuration.cs [hunk 1]
```diff
@@ -1596,6 +1596,16 @@ public partial class ProviderException : System.Exception
         public ProviderException(string message, System.Exception innerException) { }
     }
 }
+namespace System.Drawing.Configuration
+{
+    public sealed partial class SystemDrawingSection : System.Configuration.ConfigurationSection
```
Comment (weshaggard) [inline src/System.Configuration.ConfigurationManager/ref/System.Configuration.cs]: @mellinoe is this even interesting to port? It doesn't do anything so I wonder what the value of having it is.

Comment (mellinoe) [inline src/System.Configuration.ConfigurationManager/ref/System.Configuration.cs]: I don't think there's any harm in it, the whole library is for compat anyways.

**Comment by hughbe** [comment]
Test OSX x64 Debug Build

**Comment by mellinoe** [comment]
Thanks @hughbe

**Pull request merged**